book/src/developers/designs/identity-verification.md

```diff
@@ -0,0 +1,42 @@
+## Draft proposal for and Identity verification system with an out of bound challenge
```

Comment:
This should be cherry-picked out as it looks like you've got the draft doc from #1416 in this PR...?

Reply:
wops sure looks like I pushed it to the wrong branch :D